### PR TITLE
please review

### PR DESCRIPTION
**Pull request by Sezai-Keskin**

### CODE REVIEW

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Some tips about using break in switch after every case! The idea of using switch case is to make the program faster. So when we use break after every case, as soon as we faced the condition JavaScript stops to reading the other cases.
You may have already noticed the use of break for each statement. This causes the statement to 'break out' of the switch statement and continue to the next block of code. Without the break statement additional blocks may be executed if the evaluation matches the case value.
In other words when a match is found, and the job is done, if the break statement is encountered. Otherwise the switch statement will continue evaluating the expression.
You should be aware that the first match will be the statement that is executed. If you would like multiple matches to trigger code blocks then the break statement would stop that from happening.
It is not necessary to break the last case in a switch block. The block breaks (ends) there anyway.

**Review by ciamac-da — APPROVED** [review]
🥇
